examples/specs/interactive_brush.vl.json

```diff
@@ -4,7 +4,8 @@
   "data": {"url": "data/cars.json"},
   "selection": {
     "brush": {
-      "type": "interval"
+      "type": "interval",
+      "init": {"x": [55, 160], "y": [13, 37]}
```

Comment:
Shouldn't we see the brush in the SVG as well? See https://github.com/vega/vega-lite/pull/4139/files?short_path=b21ff41#diff-b21ff41954cf78b8b1141416d74085d5